sunflower-land/sunflower-land
# [FIX] Small Maneki Neko improvements

# Description

- center time left panel (resource nodes and placeables currently have time left panels centered to the top)
- fix timer display not updating for the seconds unit
- fix negative timer if both maneki nekos are shaken
- restrict clickable area to the 1x1 grid only
- remove z-indexes to avoid rendering issues with other SFTs below the Neko
- standardize pixel sizes
- prevent Nekos from disappearing when revealing and revealed prizes

![image](https://user-images.githubusercontent.com/107602352/214775650-c85d9da6-8da3-466e-b4a9-b226aac39d70.png)

## Type of change


# How Has This Been Tested?

- check countdown when having 2 nekos with different shaken at times
- hover mouse around the edges of the nekos
- click nekos to reveal prizes
- observe the nekos with/without the mouse on them for the last few seconds when the countdown is done

# Checklist:
